### PR TITLE
Add Main UI docs for integration into sidebar help

### PR DESCRIPTION
**Pull request by florian-h05**
Required for https://github.com/openhab/openhab-webui/pull/2253.

/cc @stefan-hoehn @Confectrician 

Please have a look at the CONTRIBUTING.md inside the mainui folder.

### CODE REVIEW

**Comment by florian-h05** [comment]
I have now migrated all content that was hard-coded into the UI to the docs repo and created the structure for the new Main UI docs. You can run my linked UI PR locally to see how the UI integrates the docs.

Feel free to extend the documentation, migrate docs from elsewhere in this repo and put some content into the empty files.

File: mainui/settings/scenes.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  Add Scenes using the <!--F7:blue plus_circle_fill --> plus button.
+- **Edit Scenes**<br>
+  Click on any scene in the list to edit the scene's Item commands.
+- **Delete Scenes**<br>
+  Select multiple scenes using the top **Select** button and use the **Remove** button at the bottom or use the <span style="color: red">Remove Scene</span> button on the individual scene pages.
+- **Enable/Disable Scenes**<br>
+  Use the <!--F7 pause_circle --> pause buttons on the individual scene pages.
+- **Manually run Scenes**<br>
+  Use the <!--F7:blue play --> play button on the individual scene pages to have the scene run immediately.
+<!-- END MAINUI SIDEBAR DOC - DO NOT REMOVE -->
```
Comment (stefan-hoehn) [inline mainui/settings/scenes.md:26]:
```
<!-- END MAINUI SIDEBAR DOC - DO NOT REMOVE -->

A comprehensive introduction to scenes can be found at the 
[scenes tutorial]({{base}}/tutorial/rules_scenes.html).
```

File: mainui/settings/schedule.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+# Settings - Schedule
+
+<!-- START MAINUI SIDEBAR DOC - DO NOT REMOVE -->
+View upcoming time-based rules.
+
+When you create a rule, add the `Schedule` tag and define a time trigger,
+
+![timer-trigger](../images/timer-trigger.png)
+
+this rule will be shown in the calendar view on this page.
```
Comment (stefan-hoehn) [inline mainui/settings/schedule.md:15]: then this

File: mainui/settings/schedule.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+at 7:00 in the morning
+
+![cron-seven](../images/cron-seven.png)
+
+and it will appear in the schedule view:
+
+![saturday-morning-rule](../images/saturday-rule-schedule.png)
+
+A rule that is schedule repeatedly like everyday at 8:00 in the morning will therefore be shown on every day of the calendar view.
```
Comment (stefan-hoehn) [inline mainui/settings/schedule.md:32]: A rule that is scheduled repeatedly

File: mainui/settings/scripts.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+layout: documentation
+title: Settings - Scripts
+---
+
+# Settings - Scripts
+
+<!-- START MAINUI SIDEBAR DOC - DO NOT REMOVE -->
+**Rules** are the heart of home automation.
+**Scripts** are special types of rules with only one action and no triggers, so they have to be called from "real" rules or run manually.
+They can be used to reuse code and logic across multiple other rules, one off experimentation to figure something out, create a catalog of examples, or to drive tests for other rules and behaviors.
```
Comment (stefan-hoehn) [inline mainui/settings/scripts.md:11]: for experimentation to figure

File: mainui/settings/things.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+The Things that you can add to your system are determined by the bindings that you have installed.
+
+There is a [Things concepts page]({{base}}/concepts/things.html) that explains the idea of Things in detail.
+
+On this page, you can manage all the Things you have added to your system:
+
+- **Add new Things**<br>
+  Use the [Thing Inbox](#inbox) to scan for available things or manually add ones that cannot be found automatically using the <!--F7:blue plus_circle_fill --> plus button.
+- **Configure Things**<br>
+  Click on any Thing in the list to see its configuration page and it's channels.
```
Comment (stefan-hoehn) [inline mainui/settings/things.md:19]: its channels.

File: mainui/settings/things.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+- [Adding Things - Intermediate]({{base}}/tutorial/things_intermediate.html): Things that are dealing with a binding where the bridge-Thing cannot be automatically discovered.
+- [Adding Things - Advanced]({{base}}/things_advanced.html): Things that do not support automatic discovery.
+
+## Inbox
+
+Many devices, technologies and systems can be automatically discovered on the network or browsed through some API.
+It therefore makes a lot of sense to use these features for a smart home solution.
+
+openHAB bindings can therefore implement Discovery Services for Things, which provide Discovery Results.
+All Discovery Results are put into the Thing inbox, from where they can be easily added to openHAB with just a few clicks.
```
Comment (stefan-hoehn) [inline mainui/settings/things.md:38]: Thing inbox from where they can be

File: mainui/settings/transformations.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
+---
+layout: documentation
+title: Settings - Transformations
+---
+
+# Settings - Transformations
+
+<!-- START MAINUI SIDEBAR DOC - DO NOT REMOVE -->
+**Transformations** convert a value or Item state into different value or state.
```
Comment (stefan-hoehn) [inline mainui/settings/transformations.md:9]: into a different value

**Review by stefan-hoehn — CHANGES_REQUESTED** [review]
@florian-h05 This is an amazing contribution to the docs and you earn the docs gold medal for the month 🥇 🤩

Given the amount of text you contributed I only found only a few minor issues.

File: mainui/CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+Lorem ipsum dolor sit amet, consectetur adipisici elit, ...
+<!-- END MAINUI SIDEBAR DOC - DO NOT REMOVE -->
+```
+
+## Features && Behaviour
+
+Inside these comments, the UI allows the following features:
+
+- Embed [Framework7 icons](https://framework7.io/icons/) into the UI-rendered docs using these comments:
+  - `<!--F7 $iconame -->`, where `$iconname` is a Framework7 icon's name
```
[the remark this answers is not in the document]
Comment (florian-h05) [reply mainui/CONTRIBUTING.md:25]: It is one n missing in `$iconame` ;-)

File: mainui/settings/persistence.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+# Settings - Persistence
+
+<!-- START MAINUI SIDEBAR DOC - DO NOT REMOVE -->
+**Persistence** stores data over time.
+The data may be retrieved at a later time, for example to restore your system after startup, or to prepare graphs for display on the UI.
+
+openHAB by default ships with the [RRD4J persistence service](/addons/persistence/rrd4j/) installed by default, which persists every Item on every state change and at least once a minute.
+Additionally, it restores the last stored value at system startup.
+
+To change this behaviour, you already found the correct page!
```
[the remark this answers is not in the document]
Comment (florian-h05) [reply mainui/settings/persistence.md:15]: image

File: mainui/settings/rules.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+  Click on any rule in the list to configure the rule or write a script action for the rule.
+- **Delete Rules**<br>
+  Select multiple rules using the top **Select** button and use the **Remove** button at the bottom or use the <span style="color: red">Remove Rule</span> button on the individual rule pages.
+- **Enable/Disable Rules**<br>
+  Select multiple rules using the top **Select** button and use the **Enable** or **Disable** buttons at the bottom or use <!--F7 pause_circle --> pause button on the individual rule pages.
+- **Manually run Rules**<br>
+  Use the <!--F7:blue play --> play button on the individual rule pages to have the rule run all actions immediately.
+<!-- END MAINUI SIDEBAR DOC - DO NOT REMOVE -->
+
+The page has a searchable list of rules where the following attributes
```
[the remark this answers is not in the document]
Comment (florian-h05) [reply mainui/settings/rules.md:30]: Great that you review your own contribution ;-)
I have only moved that from the old location to the new location, I haven't written this.

**Comment by florian-h05** [comment]
@stefan-hoehn Thank you for your fast review of this big PR!! I have addressed your review.

> This is an amazing contribution to the docs and you earn the docs gold medal for the month 🥇 🤩

Thank you very much, I guess I will share it with you since you wrote large parts of the docs that I moved around, and with @JustinGeorgi, who originally wrote what has been added to the UI and what I migrated to the docs repo.

**Review by stefan-hoehn — APPROVED** [review]
Thanks, looks good now.

**Comment by florian-h05** [comment]
@ghys @Confectrician Why can’t I find that documentation changes on https://next.openhab.org/docs?

**Comment by ghys** [comment]
The website fails to build on Netlify since Jan 26th:
https://app.netlify.com/sites/openhab-next/deploys/65c1516267c7c40008971007
with this error:
```
10:22:45 PM: /opt/buildhome/.rvm/rubies/ruby-2.4.3/lib/ruby/2.4.0/fileutils.rb:1225:in `stat": No such file or directory @ rb_file_s_stat - .vuepress/openhab-docs/settings/images (Errno::ENOENT)
	from /opt/buildhome/.rvm/rubies/ruby-2.4.3/lib/ruby/2.4.0/fileutils.rb:1225:in `lstat"
	from /opt/buildhome/.rvm/rubies/ruby-2.4.3/lib/ruby/2.4.0/fileutils.rb:1254:in `copy"
	from /opt/buildhome/.rvm/rubies/ruby-2.4.3/lib/ruby/2.4.0/fileutils.rb:416:in `block in copy_entry"
	from /opt/buildhome/.rvm/rubies/ruby-2.4.3/lib/ruby/2.4.0/fileutils.rb:1388:in `wrap_traverse"
	from /opt/buildhome/.rvm/rubies/ruby-2.4.3/lib/ruby/2.4.0/fileutils.rb:413:in `copy_entry"
	from /opt/buildhome/.rvm/rubies/ruby-2.4.3/lib/ruby/2.4.0/fileutils.rb:391:in `block in cp_r"
	from /opt/buildhome/.rvm/rubies/ruby-2.4.3/lib/ruby/2.4.0/fileutils.rb:1461:in `block in fu_each_src_dest"
	from /opt/buildhome/.rvm/rubies/ruby-2.4.3/lib/ruby/2.4.0/fileutils.rb:1477:in `fu_each_src_dest0"
	from /opt/buildhome/.rvm/rubies/ruby-2.4.3/lib/ruby/2.4.0/fileutils.rb:1459:in `fu_each_src_dest"
	from /opt/buildhome/.rvm/rubies/ruby-2.4.3/lib/ruby/2.4.0/fileutils.rb:390:in `cp_r"
	from prepare-docs.rb:359:in `<main>"
```
corresponding to this line in prepare-docs.rb:
https://github.com/openhab/website/blob/31c746e62fa4c3d5a3fc33c070713acfd2dd308c/prepare-docs.rb#L359

**Comment by florian-h05** [comment]
https://github.com/openhab/website/pull/445 will fix the website build.

**Pull request merged**